support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss

```diff
@@ -1,124 +1,121 @@
-// -----  gu-sass ----- //
-
-.thank-you__container .header{
-  @include mq($from: mobile, $until: tablet) {
-    background-color: gu-colour(highlight-main);
+@import './ContributionThankYouSetPassword';
+
+.thank-you__container {
+  display: flex;
+  flex-flow: column-reverse nowrap;
+  justify-content: flex-start;
+  margin: auto;
+  max-width: 1015px;
```

Comment:
do we need this? i think the parent already sets a max-width

Reply:
👍